<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>470</width>
    <height>157</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>470</width>
    <height>0</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>470</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string/>
  </property>
  <layout class="QGridLayout" name="gridLayout" rowstretch="0,1,1,0">
   <property name="horizontalSpacing">
    <number>15</number>
   </property>
   <item row="1" column="1" colspan="6">
    <widget class="QLabel" name="infoLabel">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>50</height>
      </size>
     </property>
     <property name="text">
      <string>Your Dropbox access {0}. To continue syncing, please click &lt;a href=&quot;{1}&quot;&gt;here&lt;/a&gt; to retrieve a new authorization token from Dropbox and enter it below.</string>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
     <property name="openExternalLinks">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="0" column="0" rowspan="2" alignment="Qt::AlignTop">
    <widget class="QLabel" name="iconLabel">
     <property name="minimumSize">
      <size>
       <width>60</width>
       <height>60</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>60</width>
       <height>60</height>
      </size>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="3" column="3">
    <widget class="QPushButton" name="pushButtonUnlink">
     <property name="text">
      <string>Unlink and Quit</string>
     </property>
    </widget>
   </item>
   <item row="3" column="6">
    <widget class="QPushButton" name="pushButtonLink">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>Link</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1" colspan="6">
    <widget class="QLabel" name="titleLabel">
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Expired Dropbox access</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1" colspan="6">
    <widget class="QLineEdit" name="lineEditAuthCode">
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
     <property name="placeholderText">
      <string>Authorization token</string>
     </property>
    </widget>
   </item>
   <item row="3" column="5">
    <widget class="QPushButton" name="pushButtonCancel">
     <property name="text">
      <string>Quit</string>
     </property>
    </widget>
   </item>
   <item row="3" column="4">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>1</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="3" column="1" alignment="Qt::AlignVCenter">
    <widget class="QProgressIndicator" name="progressIndicator" native="true">
     <property name="minimumSize">
      <size>
       <width>16</width>
       <height>16</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16</width>
       <height>16</height>
      </size>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QProgressIndicator</class>
   <extends>QWidget</extends>
   <header>maestral_qt.widgets.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>lineEditAuthCode</tabstop>
  <tabstop>pushButtonLink</tabstop>
  <tabstop>pushButtonCancel</tabstop>
  <tabstop>pushButtonUnlink</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
